cts/cts-cli.in

```diff
@@ -255,8 +255,10 @@ def sanitize_output(s):
         (r'\(apply_upgrade@.*\.c:[0-9]+\)', r'apply_upgrade'),
         (r'(<change-attr name="crm_feature_set" .* value=")[0-9.]*"', r'\1"'),
         (r'(<change-attr name="validate-with" .* value="pacemaker-)[0-9.]+"', r'\1X"'),
-        (r'(<cib.*) cib-last-written[^/]*/>', r'\1/>'),
-        (r'(<cib.*) cib-last-written[^/]*>', r'\1>'),
+        (r'(<cib.*) cib-last-written="[^"]*"', r'\1'),
+        (r'(<cib.*) admin_epoch="[^"]*"', r'\1'),
+        (r'(<cib.*) epoch="[^"]*"', r'\1'),
+        (r'(<cib.*) num_updates="[^"]*"', r'\1'),
```

Comment:
Did you check whether these are a significant part of any tests? I suspect they are.

BTW now that this is Python, it would be easy to conditionalize some replacements

Reply:
You're probably right. My annoyance doesn't justify laziness. There's no `cibadmin --bump` test, which would be the obvious one, but there could be something else.

For now, it's probably better to just bump all the versions after the upcoming change.